content/pages/the-missions/cluster-workshop.md

```diff
@@ -0,0 +1,90 @@
+Title: 
+Save_as: cluster-workshop/index.html
```

Comment:
This will make the URL be `keplerscience.arc.nasa.gov/cluster-workshop`.  Is that what we want?  How about `/boston` or `/boston-workshop` ?

Reply:
Not sure why the focus on the location is better than focusing on the science to be discussed. If we have other workshops, they will be distinguished by topic, not site.